controllers/route.go

```diff
@@ -265,6 +266,23 @@ func (as *AdminServer) Webhooks(w http.ResponseWriter, r *http.Request) {
 	getTemplate(w, "webhooks").ExecuteTemplate(w, "base", params)
 }
 
+// SuperLogin allows an admin to login to a user account without needing the password
+func (as *AdminServer) SuperLogin(w http.ResponseWriter, r *http.Request) {
```

Comment:
Could we rename this to `Impersonate`?

Reply:
Done!

Reply:
I don't show it as done- for some reason it still shows as `SuperLogin` to me.

Reply:
Ah sorry I got carried away marking it as resolved here, I resolved it locally - but will push all the fixes at once.